packit_service/worker/parser.py

```diff
@@ -1292,3 +1295,25 @@ def parse_pagure_pull_request_comment_event(
             comment=comment,
             comment_id=comment_id,
         )
+
+    @staticmethod
+    def parse_new_hotness_update_event(event) -> Optional[NewHotnessUpdateEvent]:
+        if "hotness.update.bug.file" not in event.get("topic", ""):
+            return None
+
+        package_name = event.get("package")
+        dg_base_url = getenv("DISTGIT_URL", PROD_DISTGIT_URL)
+
+        distgit_project_url = f"{dg_base_url}rpms/{package_name}"
+
+        version = nested_get(event, "trigger", "msg", "project", "version")
+
+        logger.info(
+            f"New hotness update event for package: {package_name}, version: {version}"
+        )
+
+        return NewHotnessUpdateEvent(
+            package_name=package_name,
+            version=version,
+            distgit_project_url=distgit_project_url,
+        )
```

Comment:
Since version and tag can be different, should we also populate the tag as well?